apis/python/tests/test_reindexer_api.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
please add at least one test that an exception is thrown if there are non-unique keys passed to build_index

Reply:
Oh, NM - I see there is a test in the other file covering this case